.travis.yml

```diff
@@ -1,38 +1,52 @@
 dist: trusty
 
+sudo: false
+
 language: php
 
-php:
-  - 5.5.9
-  - 5.6
-  - 7.0
-  - 7.1
-  - 7.2
-  - 7.3
-  - 7.4
-  - hhvm
-  - nightly
+jobs:
+  include:
+    - php: 5.5.9
+      env: LARAVEL="5.1.*"
+    - php: 7.0
+      env: LARAVEL="5.5.*"
+    - php: 7.1
+      env: LARAVEL="^5.5"
+    - php: 7.2
+      env: LARAVEL="^6.0"
+    - php: 7.2
+      env: LARAVEL="^7.0"
+    - php: 7.3
+      env: LARAVEL="^6.0"
+    - php: 7.3
+      env: LARAVEL="^7.0"
+    - php: 7.4
+      env: LARAVEL="^6.0"
+    - php: 7.4
+      env: LARAVEL="^7.0"
+    - php: nightly
+      env: LARAVEL="^6.0"
+    - php: nightly
+      env: LARAVEL="^7.0"
+    - php: hhvm-3.18
+      env: LARAVEL="^6.0"
+    - php: hhvm-3.18
+      env: LARAVEL="^7.0"
```

Comment:
Don't test laravel 6 or 7 on HHVM. It will not work. Test Laravel `5.1.*` only on HHVM. That was the last LTS to support it.